docs/workforce-management-framework/upstream-1/master.adoc

```diff
@@ -5,34 +5,41 @@ include::topics/shared/attributes.adoc[]
 :toclevels: 5
 :last-update-label!:
 
+image::../../shared/images/logo-blue.png[logo]
+
 [[Raincatcher]]
 = {WFM-RC-NameLong}
```

Comment:
@jstaffor once we move to asciibinder we may possibly resolve all the differences here. 
We may also have similar layout to yours. Going to investigate this now.